pkg/pillar/containerd/containerd.go

```diff
@@ -280,6 +595,16 @@ func (client *Client) CtrGetImage(ctx context.Context, reference string) (contai
 	return image, nil
 }
 
+func (client *Client) CtrImageExists(ctx context.Context, imageName string) (bool, error) {
```

Comment:
> Or is this to handle the non-error but does not exist case?

exactly, and since the error types come from the original containerd package I thought we should handle them here